lib/ServerChannel.js

```diff
@@ -985,18 +986,25 @@ ServerChannel.prototype.connectPhantom = function(options) {
     phantomjs.stdout.setEncoding('utf8');
     logger = Logger.get('clients', {name: 'phantomjs'});
 
-    phantomjs.stdout.on('data', function(text) {
-        logger.log(text);
+    phantomjs.stdout.on('data', function(data) {
+        logger.log('stdout: ' + data);
     });
 
-    phantomjs.on('exit', function(code) {
+    phantomjs.stderr.on('data', function(data) {
+        logger.log('stderr: ' + data, 'error');
+    });
+
+    phantomjs.on('exit', function(code, signal) {
```

Comment:
The onExit handler takes another argument, signal:
(String the signal passed to kill the child process, if it was killed by the parent.)
https://nodejs.org/api/child_process.html#child_process_event_exit